fastchat/serve/huggingface_api_worker.py

```diff
@@ -143,7 +143,10 @@ def generate_stream_gate(self, params):
         logger.info(f"gen_kwargs: {gen_kwargs}")
 
         try:
-            url = f"{self.api_base}/{self.model_path}"
+            if self.model_path == "":
```

Comment:
when can `model_path` be ""?

Reply:
the zephyr endpoint sponsored by huggingface has `""` as the model_path. I think we need to keep this.